telco-examples/mgmt-cluster/aarch64/README.md

```diff
@@ -44,7 +44,21 @@ You need to modify the following values in the `custom/scripts/99-register.sh` f
 
 You need to modify the following folder:
 
-- `base-images` - To include inside the `SL-Micro.aarch64-6.0-Default-SelfInstall-GM2.install.iso` image downloaded from the SUSE Customer Center.
+- `base-images` - To include the raw image generated by Kiwi as:
+
+```
+mkdir output
+sudo podman run --privileged -v /etc/zypp/repos.d:/micro-sdk/repos/ -v $(pwd)/output:/tmp/output -it registry.opensuse.org/isv/suse/edge/factory/images_16.0/kiwi-builder:10.2.12.0 build-image -p Default-SelfInstall
+```
```

Comment:
This should use the product image e.g `registry.suse.com/edge/3.3/kiwi-builder:10.2.12.0`